nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -54,4 +38,49 @@ For Timeouts:
 
 snippet:AzurePersistenceTimeoutsCustomization
 
-NOTE: Subscriptions and Timeouts persistence configuration only has an effect when used with Azure Storage Queues transport from NServiceBus Azure Version 6 and later.
\ No newline at end of file
+### Configuration Properties
+Each area of the persister (Sagas, Subscriptions and Timeouts) have various values that can be set or changed.
+
+NOTE: Subscriptions and Timeouts persistence configurations have no effect when used with NServiceBus.Azure Version 5 and lower.
+
+#### Saga Configuration
+  
+When using XML based configuration, the following properties can be set through the `AzureSagaPersisterConfig` section:
+
+- `ConnectionString`: Sets the connectionstring for the storage account to be used for storing saga information.  
+  - NServiceBus.Azure Versions 6 and below defaults to `UseDevelopmentStorage=true` 
+  - NServiceBus.Persistence.AzureStorage Version 1 and above defaults to `null`
+- `CreateSchema`: Instructs the persister to create the table automatically
+  - defaults to `true`
```

Comment:
Not sure I follow this. Which others are inline? All the defaults here are shown as sub lists.